completion_aggregator/models.py

```diff
@@ -205,6 +225,21 @@ def __str__(self):
             percent=self.percent,
         )
 
+    def get_values(self):
+        """
+        Return a dict object containing fields and their values to be used in bulk create or update query.
+        """
+        # these keys values will be assigned as it is
+        exact_keys = ['user', 'course_key', 'block_key', 'aggregation_name', 'earned', 'possible', 'last_modified']
+        values = {}
+        values = {key: getattr(self, key) for key in exact_keys}
+        values['user'] = values['user'].id
+        values['last_modified'] = make_datetime_timezone_unaware(values['last_modified'])
+        values['percent'] = get_percent(values['earned'], values['possible'])
+        values['created'] = make_datetime_timezone_unaware(timezone.now())
+        values['modified'] = make_datetime_timezone_unaware(timezone.now())
```

Comment:
We can't reset the value of created when updating existing `Aggregator` objects. In this method, we should only be reading the values for `self`. You can do instead:

```
values.update({key: make_datetime_timezone_unaware(getattr(self, key)) for key in [
   'last_modified', 'created', 'modified',
]})
```